Crash in ServerPreferences

#### Software and hardware versions 
Collect v1.12.0-beta.0

#### Problem description
```
Caused by: java.lang.NullPointerException: 
  at org.odk.collect.android.preferences.ServerPreferences.addPreferencesResource (ServerPreferences.java:65)
  at org.odk.collect.android.preferences.ServerPreferences.initProtocolPrefs (ServerPreferences.java:61)
  at org.odk.collect.android.preferences.ServerPreferences.onCreate (ServerPreferences.java:38)
  at android.app.Fragment.performCreate (Fragment.java:2031)
  at android.app.FragmentManagerImpl.moveToState (FragmentManager.java:863)
```

#### Steps to reproduce the problem

#### Expected behavior

#### Other information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

--- a/collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferences.java
+++ b/collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferences.java
@@ -62,7 +62,7 @@ public class ServerPreferences extends ServerPreferencesFragment implements Pref
     }
 
     private void addPreferencesResource(CharSequence value) {
-        if (value.equals(getString(R.string.protocol_odk_default))) {
+        if (value == null || value.equals(getString(R.string.protocol_odk_default))) {
             setDefaultAggregatePaths();
             addAggregatePreferences();
         } else if (value.equals(getString(R.string.protocol_google_sheets))) {

